src/OSSupport/File.cpp

```diff
@@ -453,6 +453,29 @@ AString cFile::ReadWholeFile(const AString & a_FileName)
 
 
 
+AString cFile::ReplaceFileNameInvalidChars(const AString & a_FileName, char a_Replacement)
+{
+	AString res(a_FileName);
+	for (auto & ch: res)
+	{
+		switch (ch)
+		{
+			case ':':
+			case '\\':
```

Comment:
I think we should either use an ifdef, or just substitute the colons out rather then trying to build a generic solution.